plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -247,7 +247,7 @@ bool GSDeviceOGL::Create(GSWnd* wnd)
 	m_convert.cb = new GSUniformBufferOGL(g_convert_index, sizeof(ConvertConstantBuffer));
 	// Upload once and forget about it
 	ConvertConstantBuffer cb;
-	cb.ScalingFactor = GSVector4i(theApp.GetConfig("nativeres", 0) ? 1 : theApp.GetConfig("upscale_multiplier", 2));
+	cb.ScalingFactor = GSVector4i(theApp.GetConfig("upscale_multiplier", 1)==1 || theApp.GetConfig("upscale_multiplier", 1)==7 ? 1 : theApp.GetConfig("upscale_multiplier", 2));
```

Comment:
Maybe use a local variable so you don't have to retrieve the value three times? `int upscale_multiplier = theApp.GetConfig("upscale_multiplier", 1);` ?